jobs/kube-apiserver/templates/config/bpm.yml.erb

```diff
@@ -28,6 +28,8 @@
 processes:
 - name: kube-apiserver
   executable: /var/vcap/packages/kubernetes/bin/kube-apiserver
+  limits:
```

Comment:
This is only changing the configuration for the `kube-apiserver`. It probably works on Ubuntu 14.04.

However starting Ubuntu 16.04, instead of changing `/etc/security/limits.conf`, it should change `/etc/systemd/system.conf` for the following key:

```
DefaultLimitNOFILE=65535
```

Reply:
Hi @xinzweb actually the global setting isn’t needed in this case, that’s only if systemd is launching the process.  

In the case of a bosh addon for example, the above would set the file descriptors on all systems launched parent processes to force the issue.  

Here, BPM is launching the kube-apiserver process - I’ve verified it works on Xenial.

Reply:
Thanks a lot. Glad to know that. In this case, if I want to change the existing PKS deployment with the bosh setting, do you have the pointer of what to do? Thanks.